library/Zend/Filter/Word/Service/SeparatorToSeparatorFactory.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Zend\Filter\Word\Service;
+
+use Zend\ServiceManager\Exception\ServiceNotCreatedException;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use Zend\Filter\Word\SeparatorToSeparator;
+
+class SeparatorToSeparatorFactory implements FactoryInterface
+{
+    /**
+     * @var array
+     */
+    protected $options;
+
+    /**
+     * Constructor.
+     *
+     * @param array $options
+     */
+    public function __construct(array $options = array())
```

Comment:
What happens if I try to get multiple instances of a `wordseparatortoseparator`, each with different options?